packages/inter-protocol/src/proposals/price-feed-proposal.js

```diff
@@ -242,6 +256,7 @@ export const getManifestForPriceFeed = async (
     },
   },
   installations: {
+    // ??? will every eval of price-feed-proposal will install priceAggregator ?
```

Comment:
syntax: "will" appears twice here. One of them is extraneous.

Reply:
done